jax/_src/tree_util.py

```diff
@@ -106,7 +106,11 @@ def all_leaves(iterable):
   Returns:
```

Comment:
would it be useful to also document `is_leaf` argument?

Reply:
<img width="492" alt="image" src="https://user-images.githubusercontent.com/1458824/162592177-fee52e96-8d36-4cff-9bb2-5614da5b3c66.png">

Reply:
(Done!)